### ISSUE TITLE
Failed to parse dotenv file. Encountered unexpected whitespace

### ISSUE THREAD

**Issue by vangelis183** [VemtoOrg/vemto2-issues#95]
**Error Message:**
(error) FAILED to execute command: "php" "/Applications/Vemto.app/Contents/Resources/static/VMTTL1"

**Stack:**
```
Error: Command failed: "php" "/Applications/Vemto.app/Contents/Resources/static/VMTTL1"
The environment file is invalid!
Failed to parse dotenv file. Encountered unexpected whitespace at [xxxxx xx xxx - xx].

    at ChildProcess.exithandler (node:child_process:407:12)
    at ChildProcess.emit (node:events:527:28)
    at maybeClose (node:internal/child_process:1092:16)
    at ChildProcess._handle.onexit (node:internal/child_process:302:5)
```

---

Vemto 2.0.8
darwin 23.4.0
Locale: de
App version: 2.0.8

**Comment by vangelis183** [comment]
APP_NAME with spaces results this error. 
APP_NAME was set through vemto

**Comment by TiagoSilvaPereira** [comment]
@vangelis183, thanks for reporting. This will be fixed

**Comment by TiagoSilvaPereira** [comment]
Fixed on 2.0.12
